p2020mockup/cr-6.2.8.2.html

```diff
@@ -95,7 +95,7 @@ <h2>Status of This Document</h2>
     </p>
     (@@choose one of the 2 following paragraphs)
     <p>
-      This document may be updated, replaced or obsoleted at any time. It is inappropriate to
+      This is a draft document and may be updated, replaced or obsoleted by other documents at any time. It is inappropriate to
```

Comment:
A little OCD, but Oxford comma?  
```suggestion
      This is a draft document and may be updated, replaced, or obsoleted by other documents at any time. It is inappropriate to
```

Reply:
we'll keep the comma for a future upgrade as well, since the goal here was to make the minimal amount of changes.